src/collective/frontpage/content/section.py

```diff
@@ -9,49 +9,44 @@
 from zope import schema
 from zope.interface import implementer
 from collective.frontpage import _
+from plone.namedfile.field import NamedBlobImage
 
 
 class ISection(model.Schema):
-    """ Marker interface and Dexterity Python Schema for Section
     """
+    Marker interface and Dexterity Python Schema for Section
+    """
+
+    background_type = schema.Choice(
+        title=_(u'Type'),
+        default='static',
+        vocabulary='collective.frontpage.SectionTypes',
+        required=True,
+    )
+
+    color = schema.Choice(
```

Comment:
this could be "background_color" as it will be used for backgrounds only, as far as I know. (Maybe @santonelli can give a comment about this)